libx509pq.c

```diff
@@ -22,6 +22,7 @@
 #include "access/htup_details.h"
 #include "utils/timestamp.h"
 #include "utils/builtins.h"
+#include "assert.h"
```

Comment:
"man 3 assert" says "assert - abort the program if assertion is false".  I don't think libx509pq should ever be killing postgres processes.